src/app/components/book/bookController.js

```diff
@@ -0,0 +1,40 @@
+angular.module('app').controller('BookController', ['bookService',
+function(bookService) {
+
+  this.main = {
```

Comment:
Don't attach local variables to the controller scope. Declare them using let or const accordingly. Only variables or method that will be used in the view are declared in the controller scope.